metcalcpy/calc_difficulty_index.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+"""
+Program Name: calc_difficulty_index.py
+
+Forecast decision difficulty indices.
+
+Implement a set of decision difficulty indices for forecasts of postive
+definite quantities such as wind speed and wave height.
+
+Created on Thu Feb 20 16:23:06 2020
+Last modified on Mon Mar 30 11:25:30 2020
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from metcalcpy.piecewise_linear import PiecewiseLinear as plin
+
+__author__ = 'Bill Campbell (NRL) and Lindsay Blank (NCAR)'
+__version__ = '0.1.0'
+__email__ = 'met_help@ucar.edu'
+
+# Enforce positive definiteness of quantities such as standard deviations
+EPS = np.finfo(np.float32).eps
+# Only allow 2D fields for now
+FIELD_DIM = 2
+
+def _input_check(sigmaij, muij, threshold, fieldijn, sigma_over_mu_ref, under_factor):
+    """
+    Check for valid input to _difficulty_index.
+
+    Parameters
+    ----------
+    sigmaij : 2D numpy array
+        Positive definite array of standard deviations of a 2D field.
+    muij : Float scalar or 2D numpy array
+        The mean values corresponding to sigmaij.
+    threshold : Float (or int) scalar
+        A significant value to be compared with values of the forecast field
+        for each ensemble member.
+    sigma_over_mu_ref : Scalar
+    fieldijn : 3D numpy array
+        Values of the forecast field. Third dimension is ensemble member.
+    under_factor : Float scalar
+        Must be between 0 and 1.
+
+    Returns
+    -------
+    None.
+
+    """
+    print("We made it here")

```

Comment:
Probably a good idea to get rid of this print statement

Reply:
It would help if I got rid of my inelegant debugging strings. I'll delete those, push, and then merge. Thanks, Minna!